Adyen/Core/Components/AbstractPersonalInformationComponent/AbstractPersonalInformationComponent+Extensions.swift

```diff
@@ -29,6 +29,12 @@ extension AbstractPersonalInformationComponent: LoadingComponent {
 
 extension AbstractPersonalInformationComponent: TrackableComponent {
 
+    /// :nodoc
+    public func viewDidLoad(viewController: UIViewController) {
+        let flavor: TelemetryFlavor = _isDropIn ? .dropInComponent : .components(type: paymentMethod.type)
+        adyenContext.analyticsProvider.trackTelemetryEvent(flavor: flavor)
+    }
```

Comment:
should it be `viewDidLoad` or `viewWillAppear`? 🤔

Reply:
It's doesn't really matter in reality. I'm following the same behavior we have for the 1px thing. I guess it's better to send it as soon as the component is being loaded.

Reply:
Also, on the web they're sending the event before thew view gets painted.

Reply:
Sending the telemetry event when the view will appear.